bucket/lagrange.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "1.2.2",
+    "description": "A Beautiful Gemini Client",
+    "homepage": "https://gmi.skyjake.fi/lagrange",
+    "license": "BSD-2-Clause",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/skyjake/lagrange/releases/download/v1.2.2/lagrange_v1.2.2_windows-x64_portable.zip",
+            "hash": "6e86c6b4691167ae31824fc043d7175a984dae572cf2bfa4d140bd4b7cce8379"
+        }
+    },
+    "bin": "lagrange.exe",
```

Comment:
There are no command line parameters available from quick look and the command line is stuck until the window is presented

![image](https://user-images.githubusercontent.com/13260377/110999474-b6271800-8380-11eb-915c-ed241e95ccbe.png)


```suggestion
```

Reply:
![image](https://user-images.githubusercontent.com/51169564/111000216-a0e5c580-834f-11eb-90f6-0e8d6257a716.png)